### PR TITLE
Exception message review by Shay

### PR DESCRIPTION
**Pull request by smitpatel**

### CODE REVIEW

File: src/EFCore.Relational/Properties/RelationalStrings.resx [hunk 1]
```diff
@@ -478,6 +478,45 @@
   <data name="InvalidMinBatchSize" xml:space="preserve">
     <value>The specified 'MinBatchSize' value '{value}' is not valid. It must be a positive number.</value>
   </data>
+  <data name="JsonEntityMappedToDifferentViewThanOwner" xml:space="preserve">
+    <value>Entity '{jsonType}' is mapped to JSON and also mapped to a view '{viewName}', but its owner '{ownerType}' is mapped to a different view '{ownerViewName}'. Every entity mapped to JSON must also map to the same view as its owner.</value>
```
Comment (roji) [inline src/EFCore.Relational/Properties/RelationalStrings.resx:482]:
```suggestion
    <value>Entity '{jsonType}' is mapped to JSON and also to a view '{viewName}', but its owner '{ownerType}' is mapped to a different view '{ownerViewName}'. Every entity mapped to JSON must also map to the same view as its owner.</value>
```

File: src/EFCore.Relational/Properties/RelationalStrings.resx [hunk 2]
```diff
@@ -478,6 +478,45 @@
...
+  </data>
+  <data name="JsonEntityMultipleRootsMappedToTheSameJsonColumn" xml:space="preserve">
+    <value>Multiple owned root entities are mapped to the same JSON column '{column}' in table '{table}'. Each owned root entity must map to a different column.</value>
+  </data>
+  <data name="JsonEntityOwnedByNonJsonOwnedType" xml:space="preserve">
+    <value>Owned entity type '{nonJsonType}' is mapped to table '{table}' and contains JSON columns. This is currently not supported. All owned types containing a JSON column must be mapped to a JSON column themselves.</value>
+  </data>
+  <data name="JsonEntityReferencingRegularEntity" xml:space="preserve">
+    <value>Entity type '{jsonEntity}' is mapped to JSON and has navigation to a regular entity which is not the owner.</value>
```
Comment (roji) [inline src/EFCore.Relational/Properties/RelationalStrings.resx:491]:
```suggestion
    <value>Entity type '{jsonEntity}' is mapped to JSON and has a navigation to a regular entity which is not the owner.</value>
```

File: src/EFCore.Relational/Properties/RelationalStrings.resx [hunk 3]
```diff
@@ -478,6 +478,45 @@
...
+    <value>Entity type '{jsonEntity}' is part of a collection mapped to JSON and has its ordinal key defined explicitly. Only implicitly defined ordinal keys are supported.</value>
+  </data>
+  <data name="JsonEntityWithIncorrectNumberOfKeyProperties" xml:space="preserve">
+    <value>Entity type '{jsonEntity}' has an incorrect number of primary key properties. Expected number is: {expectedCount}, actual number is: {actualCount}.</value>
+  </data>
+  <data name="JsonEntityWithMultiplePropertiesMappedToSameJsonProperty" xml:space="preserve">
+    <value>Entity '{jsonEntity}' is mapped to JSON and contains multiple properties or navigations which are mapped to the same JSON property '{property}'. Each property should map to a unique JSON property.</value>
+  </data>
+  <data name="JsonEntityWithNonTphInheritanceOnOwner" xml:space="preserve">
+    <value>Entity type '{rootType}' references entities mapped to JSON. Only '{tph}' inheritance is supported for those entities.</value>
```
Comment (roji) [inline src/EFCore.Relational/Properties/RelationalStrings.resx:509]: Is the variable '{tph}' correct here, should it just be literal TPH (as the only supported option)?

**Review by roji — APPROVED** [review]
A couple of extra minor fixes, thanks.

File: src/EFCore.Relational/Properties/RelationalStrings.resx (hunk 3 above)
Comment (smitpatel) [reply src/EFCore.Relational/Properties/RelationalStrings.resx:509]: I don't know. cc: @AndriySvyryd @maumar

Comment (AndriySvyryd) [reply src/EFCore.Relational/Properties/RelationalStrings.resx:509]: inline it

**Pull request merged**